### ISSUE TITLE
Come up with a way to easily copy the links

### ISSUE THREAD

**Issue by JoshKoiro** [JoshKoiro/Nautical-Net#2]
Maybe should be able to just click on the embed? or right click? Need to refer to the Discord API documentation to get some options of how to use the interface.

**Comment by JoshKoiro** [comment]
It looks like the first step is to set the embed `setURL()` method to equal the URL of the image.

next, we will have to use the clipboard object from the 'electron' module:
```
npm install electron --save
```

the main method we are looking to use is clipboard.writeText(URL)

Once this is done, perhaps we want to send some kind of notification or something.

**Comment by JoshKoiro** [comment]
Not sure if I will be able to do this or not with discord. Might push this off to be something lower priority.

**Comment by JoshKoiro** [comment]
Going to close this for now - maybe pick this up at a future date.
